src/runtime/runtime.cs

```diff
@@ -1966,6 +1985,15 @@ internal static IntPtr PyMem_Realloc(IntPtr ptr, long size)
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern void PyErr_Print();
 
+        //====================================================================
+        // Cell API
+        //====================================================================
+
+        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern NewReference PyCell_Get(IntPtr cell);
+
+        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern int PyCell_Set(IntPtr cell, IntPtr value);
```

Comment:
NIT: you can use `BorrowedReference` for arguments, that take a `PyObject*` and don't steal a reference.